help_test.go

```diff
@@ -101,6 +101,39 @@ func Test_Help_Custom_Flags(t *testing.T) {
 	}
 }
 
+func Test_Help_Required_Flags_Does_Not_Error(t *testing.T) {
+	oldFlag := HelpFlag
```

Comment:
It doesn't look like we actually need to save/reset this, since the test doesn't modify it